docs/settings.rst

```diff
@@ -75,6 +75,10 @@ tokens submitted to the refresh view will be added to the blacklist.
 When set to ``True``, causes refresh tokens submitted to the
 ``TokenRefreshView`` to be added to the blacklist if the blacklist app is in
 use and the ``ROTATE_REFRESH_TOKENS`` setting is set to ``True``.
+You need to add ``'rest_framework_simplejwt.token_blacklist',`` to your 
+``INSTALLED_APPS`` in the settings file to use this settings.
```

Comment:
There's a little typo here.  It should be "...in the settings file to use this setting."  In your version, the word "settings" at the end of the sentence is incorrectly pluralized.

Reply:
@davesque :+1:  https://github.com/SimpleJWT/django-rest-framework-simplejwt/pull/310